### ISSUE TITLE
selecting Run without Debugging is still trying to debug

### ISSUE THREAD

**Issue by atriche** [microsoft/vscode#234834]
Type: <b>Bug</b>

Selecting Run without Debugging or using the hotkey will still try to debug. Just installed VScode. 

VS Code version: Code 1.95.3 (f1a4fb101478ce6ec82fe9627c43efbf9e98c813, 2024-11-13T14:50:04.152Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 9 3900X 12-Core Processor             (24 x 3800)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.93GB (8.10GB free)|
|Process Argv|--crash-reporter-id ffd60c33-7df0-4f21-9479-4d2f78f12241|
|Screen Reader|no|
|VM|0%|
</details>Extensions: none<details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805cf:30301675
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyonecf:30548226
962ge761:30959799
pythonnoceb:30805159
asynctok:30898717
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
da93g388:31013173
dvdeprecation:31068756
dwnewjupytercf:31046870
nativerepl1:31139838
pythonrstrctxt:31112756
nativeloc1:31185841
cf971741:31144450
iacca1:31171482
notype1cf:31157160
5fd0e150:31155592
dwcopilot:31170013
stablechunks:31184530

```

</details>

**Comment by gjsjohnmurray** [comment]
What language are you coding in?

**Comment by atriche** [comment]
I used Javascript - I was following the VSCode basics video. The guy just types a line and hits f5 and his code runs.... but I am learning that is NOT how vscode works. So I think that video needs an update.
https://youtu.be/B-s71n0dHUk?si=N27KimdKgljKS0RU

I found an extension that allows me to run my code the way I see in the video (but the video makes no mention of needing anything else after a new install).
